olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeeventing_crd.yaml

```diff
@@ -1133,6 +1133,20 @@ spec:
                             type: object
                         type: object
                       type: array
+              namespace:
+                description: A field of namespace name to override the labels and annotations
+                type: object
+                properties:
+                  labels:
+                    additionalProperties:
+                      type: string
+                    description: Labels overrides labels for the namespace and its template.
+                    type: object
+                  annotations:
+                    additionalProperties:
+                      type: string
+                    description: Annotations overrides labels for the namespace and its template.
+                    type: object
```

Comment:
@pierDipi @skonto `namespace` is now added in both Serving and Eventing CRDs. PTAL if it shouldn't be dropped.

Reply:
Introduced here: https://github.com/knative/operator/commit/36698ccd6a79078f20229fafc18dd514bbaa100b#diff-06b711a80cdfd8efc0e8ecd20d9741e80f959114d68dc44e8faaf01c5ab0d7f9

Reply:
I am not sure we need this since we only install in our own dedicated ns.